UmbracoFood.Infrastructure/Mapping/RestaurantRepositoryMappingProfile.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AutoMapper;
+using UmbracoFood.Core.Models;
+using UmbracoFood.Infrastructure.Models.POCO;
+
+namespace UmbracoFood.Infrastructure.Mapping
+{
+    public class RestaurantRepositoryMappingProfile : Profile
+    {
+        protected override void Configure()
+        {
+            CreateMap<RestaurantPoco, Restaurant>()
```

Comment:
https://automapper.codeplex.com/wikipage?title=Configuration%20Validation

Można dać coś takiego
Mapper.AssertConfigurationIsValid();
Ale to opłaca się dać w projecie UmbracoFood.Tests :) Bo to sprawdzi Wszystkie zarejestrowane profile